src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Filters/ShapeRenderFilter.cs

```diff
@@ -32,6 +33,10 @@ static async ValueTask<FluidValue> Awaited(Task<IHtmlContent> task)
 
                 return new ValueTask<FluidValue>(new HtmlContentValue(task.Result));
             }
+            else if (obj is IHtmlContent html)
```

Comment:
Are you sure about this? Should it not be (if obj is IShape && obj is IHtmlContent) at the same time?

Reply:
Don't think so. basicaly `Shape` is not `IHtmlContent`

It should be equivalent to DisplayAsync ?https://github.com/OrchardCMS/OrchardCore/blob/37459b27a7437d398998e521a650a65172db6b3e/src/OrchardCore/OrchardCore.DisplayManagement/Razor/RazorPage.cs#L82-L100

Maybe we should also handle strings.

Reply:
This example is actually only kept for backward compatibility. We will remove `DisplayAsync(dynamic)` in a later version, to keep the strongly types one.